internal/input/report.go

```diff
@@ -0,0 +1,204 @@
+// Copyright The Enterprise Contract Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.`
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package input
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	ecc "github.com/enterprise-contract/enterprise-contract-controller/api/v1alpha1"
+	"github.com/hashicorp/go-multierror"
+	"gopkg.in/yaml.v2"
```

Comment:
Weird that this works! At some point it didn't handle json struct annotations like `sigs.k8s.io/yaml` does.

Reply:
removed this as I switched to using `sigs.k8s.io/yaml`